dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -17,15 +17,43 @@
 #include "utils/datetime.h"
 #include "utils/fileutils.h"
 #include "utils/signal-monitor.h"
+#include "utils/stringutils.h"
+
+typedef std::string string;
+
+static struct RumContext {
+    string application_id;
+    string session_id;
+    string view_id;
+
+    RumContext() : application_id(""), session_id(""), view_id("") {}
```

Comment:
nit: with the type `string`, you don't need to pass the empty string explicitly, `application_id()` is enough